docs/api/version-history.md

```diff
@@ -94,6 +94,11 @@ keywords: "API, Docker, rcli, REST, documentation"
   actually supported. API versions before v1.42 continue to ignore these parameters
   and default to attaching to all streams. To preserve the pre-v1.42 behavior,
   set all three query parameters (`?stdin=1,stdout=1,stderr=1`).
+* `/containers/{name}/attach/ws` endpoint only attach to configured streams
```

Comment:
Oh! Looks like a bad rebase; this bullet is unrelated

Reply:
Whops, my bad. Fixed!